controllers/keyword.go

```diff
@@ -0,0 +1,51 @@
+package controllers
+
+import (
+	"net/http"
+
+	. "google-scraper/helpers"
+	. "google-scraper/models"
+	"google-scraper/presenters"
+
+	"github.com/beego/beego/v2/core/logs"
+	"github.com/beego/beego/v2/server/web"
+)
+
+type KeywordController struct {
+	baseController
+}
+
+func (c *KeywordController) Show() {
+	web.ReadFromRequest(&c.Controller)
+	keyword, err := c.findKeyword()
+	if err != nil {
+		flash := web.NewFlash()
+		flash.Error(err.Error())
+		flash.Store(&c.Controller)
+		c.Ctx.Redirect(http.StatusFound, "/")
+		return
+	}
+
+	keywordPresenter, err := presenters.InitializeKeywordPresenter(keyword)
+	if err != nil {
+		logs.Error("Initializing presenter failed: ", err)
+	}
+
+	c.setAttributes(keywordPresenter)
+}
+
+func (c *KeywordController) findKeyword() (*Keyword, error) {
+	keywordId := c.Ctx.Input.Param(":id")
+	Id, err := StringToInt(keywordId)
```

Comment:
@junan Should we omit the way to cast string to integer?
I think we can still get the record 🤔
So we don't have to handle with this error.
What do you think?

<img width="600" alt="Screen Shot 2564-03-25 at 10 33 20" src="https://user-images.githubusercontent.com/749672/112414949-8f3fed00-8d55-11eb-91f0-a0cd35825e7d.png">

Reply:
Ya, It can work without casting also. But converting to integer I think a bit improvement because `ID` generated expected to be an `integer` and 

- I dont need to pass `Id` as string in `FindKeywordBy(..)`
- While testing `FindKeywordBy(..)` I can pass interger Id istead of string

Reply:
I just want to reduce with error checking, if you want to keep like this I'm still OK with that